### ISSUE TITLE
Update MAAP Documentation

### ISSUE THREAD

**Issue by marjo-luc** [MAAP-Project/ZenHub#534]
Once new extensions have been deployed, update MAAP documentation to remove references to extensions that are no longer used.

- [x] Repo readmes

- [x]  MAAP Project Read the Docs

- [ ]  ~Add contributing guidelines for extension repos~

- [x]  Confirm extension repos have proper licenses (Apache 2.0)

**Comment by marjo-luc** [comment]
The readmes for the new extensions have been updated. I've also enabled the github discussion boards for each of the extension repos. See [here](https://github.com/MAAP-Project/dps-jupyter-extension/discussions) for an example. I'm hoping this will be a more dev-centric venue that will foster better communication amongst the community by providing extension FAQ's, discussion for new features, and release announcements with user-friendly change logs.
